app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -1204,3 +1204,9 @@ select#aspect_ids_ {
     background-color: $light-green;
   }
 }
+
+/* --- Tag page --- */
+#app #main h1 {
+  overflow: hidden;
```

Comment:
Please remove this line. Your fix also works without it and this line breaks the style of tags that contain letters like 'g':
![broken tag style](https://cloud.githubusercontent.com/assets/3798614/6931409/e970295c-d80c-11e4-8821-bc1d4b5333cb.png)